kiwi/volume_manager/lvm.py

```diff
@@ -188,7 +199,7 @@ def create_volumes(self, filesystem_name):
             full_size_volume = canonical_volume_list.full_size_volume
             log.info('--> fullsize volume %s', full_size_volume.name)
             self._create_volume_no_zero(
-                [
+                self.lvm_tool_options, [
```

Comment:
same here

Reply:
```python
@staticmethod
    def _create_volume_no_zero(lvm_tool_options, lvcreate_args):
        ...
```

the private helper method gets self.lvm_tool_options as argument. I don't see an error ?